meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: "pytest-playwright"
+  version: "{{ environ.get('GIT_DESCRIBE_TAG') }}"
+
+source:
+  path: .
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - playwright >=v1.33.0
```

Comment:
The way playwright-python (and now playwright-pytest) create their conda versions from the v prefixed version git tags makes this look a little funny (and tripped me up for a while in testing) as it isn't very consistent with the pure semantic version number or date version number most other conda packages use. Not sure whether you might want to exclude that v prefix at some point.

Reply:
Good finding! Lets remove the v prefix directly for the pytest plugin and we can follow-up with a fix for the normal `playwright` plugin. Do you think this is a breaking change? SemVer should behave similar in both cases I feel like.